packages/react/src/Pagination/Pagination.test.tsx

```diff
@@ -53,6 +54,20 @@ describe('Pagination', () => {
     expect(linksAsVerbatimText).toEqual(['Previous', '1', '…', '3', '4', '5', '6', '7', '…', '10', 'Next'])
   })
 
+  it('applies the correct attributes to the ellipsis button', () => {
+    const {container} = render(<Pagination pageCount={10} currentPage={1} />)
+
+    // Testing library can't find this element as it's presentation-only, so we need to use querySelector
```

Comment:
Could you elaborate on this part please? Wondering why Testing Library can't see it, but a query selector can. They should be capable of doing similar things. 

Did `getByRole('presentation')` or `queryByRole('presentation')` not work?

Reply:
It was a while back so I'm not 100% sure why I did this 😅

I've just done some testing and I think that I'd (incorrectly) assumed that testing library would ignore elements with a role of "presentation" in the same way it ignores hidden elements.

That doesn't seem to be the case though, so I'll swap this out for a `getByRole`, thanks!

Reply:
I may have tried selecting it using `getByRole('presentation', {name: '…'})` which _doesn't_ work, so possibly that's where my incorrect assumption came from